pkg/templates/templates.go

```diff
@@ -54,11 +59,12 @@ type TemplateResolver struct {
 	// Required
 	kubeClient *kubernetes.Interface
 	// Optional
-	kubeAPIResourceList []*metav1.APIResourceList
-	kubeConfig          *rest.Config
-	lookupNamespace     string
-	startDelim          string
-	stopDelim           string
+	AdditionalIndentation uint
+	kubeAPIResourceList   []*metav1.APIResourceList
+	kubeConfig            *rest.Config
+	lookupNamespace       string
+	startDelim            string
+	stopDelim             string
```

Comment:
@mprahl : Could you please remind me why we two sets of the same params in TemplateResolver and Config. Couldn't we have a config struct in TemplateResolver.

Reply:
I was wondering the same thing, but I can't think of a good reason. Since that is outside of the scope of this PR, I'd like that change to be separate.

Reply:
ok, could you pls make one for that as well and i will approve both together

Reply:
@ckandag I'd rather not hold this PR hostage based on tech debt reduction work that doesn't affect the user. I'm pretty busy with other work at the moment.

Could you please review this one separately and if I have time, I'll refactor it in another PR.